src/pods/toolbar/components/save-button/save-button.component.tsx

```diff
@@ -49,6 +49,7 @@ export const SaveButton = () => {
       label={'Save'}
       onClick={() => saveFile(openedFilename)}
       className={classes.button}
+      disabled={!canvasSchema.tables.length}
```

Comment:
Hello @SkayDks, the check should include that length >= 1. We want to enable this button when there is at least one collection in the model